src/worker/sandbox/dockerPool.go

```diff
@@ -94,19 +105,33 @@ func (pool *DockerPool) Create(parent Sandbox, isLeaf bool, codeDir, scratchDir
 		pkgDirs = append(pkgDirs, "/packages/"+pkg+"/files")
 	}
 
+	// create the container using the specified configuration
+	procLimit := int64(common.Conf.Limits.Procs)
+	swappiness := int64(common.Conf.Limits.Swappiness)
+	cpuPercent := int64(common.Conf.Limits.CPU_percent)
+	var memoryLimit int64
+	if strings.Contains(scratchDir, "scratch") {
```

Comment:
Let's use the meta.MemLimitMB to decide the memory limit.  That should be specified at a higher level, like here: https://github.com/open-lambda/open-lambda/blob/main/src/worker/lambda/packages/packagePuller.go#L180.